tests/components/matter/test_sensor.py

```diff
@@ -250,6 +260,24 @@ async def test_battery_sensor(
     assert entry
     assert entry.entity_category == EntityCategory.DIAGNOSTIC
 
+    """Test battery voltage sensor."""
+    entity_id = "sensor.eve_door_battery_voltage"
+    state = hass.states.get(entity_id)
+    assert state
+    assert state.state == "3558"
+
+    set_node_attribute(eve_contact_sensor_node, 1, 47, 11, 100)
```

Comment:
Update the BatVoltage attribute. It won't have the same integer id as BatPercentRemaining. Find the correct id number in the fixture. Then change the value to something appropriate in the mV range.

Reply:
Yes _battery_voltage_ is attribute _11_
`1/47/11": 3558,`